big_tests/rebar.config

```diff
@@ -1,7 +1,7 @@
 {erl_opts, [debug_info,
             {i, ["include"]}]}.
 
-{require_otp_vsn, "(1[89])|20"}.
+{require_otp_vsn, "(1[89])|2[01]"}.
```

Comment:
`19|2[01]`?

Reply:
Or eve, `19|20|21` it will be easier to understand. I'll change that.